solaris/solaris11/generate_wazuh_packages.sh

```diff
@@ -72,9 +72,11 @@ check_version(){
     if [ "${major_version}" -eq "3" ]; then
         if [ "${minor_version}" -ge "5" ]; then
             deps_version="true"
+            disa_share="true"
```

Comment:
This line is disabling the compilation of the `.so` file to 3.5.0 or greater. This should be only applied if the script is building a package greater or equal to v3.10.0.

Reply:
Change